### PR TITLE
feat(components-native): Import ErrorMessageWrapper to components-native

### PR DESCRIPTION
**Pull request by rodrigoeidelvein**
### Added

- `ErrorMessageWrapper` component in components-native

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components-native/src/ErrorMessageWrapper/ErrorMessageWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+const wrapForStyle: Record<WrapForTypes, ViewStyle | undefined> = {
+  card: styles.wrapForCard,
+  default: undefined,
+};
+
+/**
+ * Adds an error message below the children but ensure the message gets read
+ * out first.
+ *
+ * This component is internal to Atlantis and shouldn't be used outside of it.
```
Comment (taylorvnoj) [inline packages/components-native/src/ErrorMessageWrapper/ErrorMessageWrapper.tsx:34]: This comment... scares me haha. Do we know the context behind it?

**Review by taylorvnoj — COMMENTED** [review]
Just a question about a comment...

File: packages/components-native/src/ErrorMessageWrapper/ErrorMessageWrapper.tsx (hunk 1 above)
Comment (rodrigoeidelvein) [reply packages/components-native/src/ErrorMessageWrapper/ErrorMessageWrapper.tsx:34]: Probably the same thing as `Typography`. It is an Atlantis component, but should be only be used internally. Even though I noticed that there's a component in jobber-mobile using it 🤔

Comment (taylorvnoj) [reply packages/components-native/src/ErrorMessageWrapper/ErrorMessageWrapper.tsx:34]: Right - in `PaymentMethodCreateOverlay` right?

There is [this comment](https://github.com/GetJobber/jobber-mobile/pull/4456#discussion_r935667449) from @darryltec  original PR - any updated thoughts on that @rodrigoeidelvein @darryltec ?

Comment (darryltec) [reply packages/components-native/src/ErrorMessageWrapper/ErrorMessageWrapper.tsx:34]: I mean, the people can use it if they understand what it's doing

**Review by taylorvnoj — APPROVED** [review]
LGTM

**Pull request merged**